### DISCUSSION TITLE
Which Distro/Desktopmanager for a Surface Pro 9?

### DISCUSSION THREAD

**Discussion by champagnierle** [linux-surface/linux-surface#1291] [General]
Hi,
I'm working with Debian since Suse 2.4 now. On the desktops and on the laptops this is working really well. I'm used to working with Debian stable (except my own device) as it is exactly as the name. 
I bought a Surface Pro for my music for having no more trouble with too much sheets on the music stand..

I'm at the beginning of the tests and I thought Gnome would be a good decision. But my Surface gets a bit more than just warm when browsing the net.

Therefore I'm wondering if this is due to too much background jobs. I'm used to using fluxbox on my device but I think there is no support for tablet devices. I tried LXDE but I don't achieve in integrating the virtual keyboard when using without the attached one.

Do you have any hints for me?

Best regards and thanks for all replies

Marc

**Comment by shafai** [comment]
I have SP9 with ArchLinux+Gnome and mine does not get hot. I have serious issues with suspend/hibernate though. does yours drain the battery when in suspend and/or hibernate?

Ali

**Comment by champagnierle** [comment]
Hi,
I missed to tell a lot of things... I'm using an SP9 with i5. Still on debian, but I tried a lot yesterday. 
As the SP9 has a HiDPI display the right parameters for a readable presentation are very important. I found out that an actual gnome doesn't make too much fun. Gnome Classic on wayland uses the HiDpi in a good way. For presenting the sheet music I'm scanning the sheets to pdf and present them with oKular.
I tried KDE which I like more but KDE on wayland has a good presentation but it cannot be used undocked as there is absolutely no virtual keyboard...
I tried fluxbox and LXDE but both failed with the display. If you want to use a shell under a microscope it is worth a try, otherwise just log out and look for something on wayland....
Right now I'm on classic gnome on wayland... Or if it doesn't work properly I still have the W11 on dual boot. But I would really like to make Linux work.
Right now I don't see battery drain. S2Disk is not yet configured but s2Ram works and I think that I didin't lose any battery power.

I really love the SP9. It is a really multifunctional device with a great presentation, a very good display and even cheaper than a 12.9'' IPad...

Something I also forgot (shame on me) to tell yesterday:
THANKS TO ALL who make linux-on-surface possible!

Best regards from southern Germany

Marc
P.S.: Here's a link to the webcam of my home town:  [https://singen.panomax.com](https://singen.panomax.com) !
